tests/NRedisStack.Tests/Json/JsonTests.cs

```diff
@@ -775,11 +774,49 @@ await commands.MSetAsync(values)
         await Assert.ThrowsAsync<ArgumentOutOfRangeException>(async () => await commands.MSetAsync(new KeyValuePath[0]));
     }
 
+    [Fact]
+    [Trait("Category", "edge")]
+    public void Merge()
+    {
+        IJsonCommands commands = new JsonCommands(redisFixture.Redis.GetDatabase());
+
+        // Create a key
+        Assert.True(commands.Set("test_merge", "$", "{\"a\":{\"b\":{\"c\":\"d\"}}}"));
```

Comment:
Is there an easier way to construct a JSON string.. some sort of raw, multiline string?.
It would make test easier to read for others. Perhaps also with real values instead of just letters.